src/components/Documentation/Layout/index.tsx

```diff
@@ -13,6 +13,14 @@ import styles from './styles.module.css'
 const Layout: LayoutComponent = ({ children, ...restProps }) => {
```

Comment:
where do we get `restProps` ? can we make this trailing slash issue localized somehow? (don't like that we have to apply "slice" in a few places already

Reply:
You're right to be suspect of this check. I was thinking that the `location` from here is the one Gatsby provides, but that wouldn't change with the URL so this must be the global `location` from the browser that's prop drilled into the docs component for some reason.

Next push will have a change that resolves this.

Reply:
I've now changed the way the Layout sources the current path:

before, it came from the window global `location`, but now it comes from Gatsby's static `location`. Since the paths Gatsby provides are all ensured to have the trailing slash removed, we now don't have to check for it- at least for anything that sources `pathname` from the Layout, and I think I got every instance of that here.